test/t/matches/expired_match_test.exs

```diff
@@ -22,50 +22,10 @@ defmodule T.Matches.ExpiredMatchTest do
       assert Matches.expiration_list_expired_matches(_at = ~U[2021-01-03 12:00:00Z]) == []
       assert Matches.expiration_list_expired_matches(_at = ~U[2021-01-10 12:00:00Z]) == []
     end
-
-    test "lists matches with no voicemail exchange after two days" do
-      match = match(inserted_at: ~N[2021-01-01 12:00:00])
-
-      assert Matches.expiration_list_expired_matches(_at = ~U[2021-01-03 12:00:00Z]) == []
-
-      assert Matches.expiration_list_expired_matches(_at = ~U[2021-01-03 12:00:01Z]) ==
-               [%{id: match.id, user_id_1: match.user_id_1, user_id_2: match.user_id_2}]
-    end
-
-    test "lists matches with one-sided voicemail exchange after two days" do
-      match = match(inserted_at: ~N[2021-01-01 12:00:00])
-      voicemail(match: match, caller_id: match.user_id_1)
-      voicemail(match: match, caller_id: match.user_id_1)
-
-      assert Matches.expiration_list_expired_matches(_at = ~U[2021-01-03 12:00:00Z]) == []
-
-      assert Matches.expiration_list_expired_matches(_at = ~U[2021-01-03 12:00:01Z]) ==
-               [%{id: match.id, user_id_1: match.user_id_1, user_id_2: match.user_id_2}]
-    end
-
-    test "lists matches with voicemail exchange after seven days" do
-      match = match(inserted_at: ~N[2021-01-01 12:00:00])
-
-      voicemail(match: match, caller_id: match.user_id_1)
-      voicemail(match: match, caller_id: match.user_id_1)
-      voicemail(match: match, caller_id: match.user_id_2)
-
-      assert Matches.expiration_list_expired_matches(_at = ~U[2021-01-08 12:00:00Z]) == []
-
-      assert Matches.expiration_list_expired_matches(_at = ~U[2021-01-08 12:00:01Z]) ==
-               [%{id: match.id, user_id_1: match.user_id_1, user_id_2: match.user_id_2}]
-    end
   end
 
   describe "expiration_list_soon_to_expire/0,1" do
-    defp in_pre_voicemail_exp_notification_window(match) do
-      match.inserted_at
-      |> DateTime.from_naive!("Etc/UTC")
-      |> DateTime.add(Matches.pre_voicemail_ttl())
-      |> DateTime.add(_3h = -3 * 3600)
-    end
-
-    defp in_post_voicemail_exp_notification_window(match) do
+    defp exp_notification_window(match) do
```

Comment:
```suggestion
    defp in_exp_notification_window(match) do
```

Reply:
since the function returns a single value in the window, not the window itself, which would be a range